README.md

```diff
@@ -31,13 +31,14 @@ Available as [PyPI module](https://pypi.python.org/pypi/disposable-email-domains
 True
 ```
 
-**PHP** contributed by @txt3rob and @deguif
+**PHP** contributed by [@txt3rob](https://github.com/txt3rob), [@deguif](https://github.com/deguif) and [@pjebs](https://github.com/pjebs)
 ```php
-function is_temp_mail($mail) {
-    $mail_domains_ko = file('disposable_email_blacklist.conf', FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
-
-    //Need to ensure the mail contains an @ to avoid undefined offset
-    return in_array(explode('@', $mail)[1], $mail_domains_ko);
+function is_disposable_email($email) {
+  $path = realpath(dirname(__FILE__)) . '/disposable_email_blacklist.conf';
+  $mail_domains_ko = file($path, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
+  $mail_domains_ko = array_fill_keys($mail_domains_ko, true);
+  $domain = mb_strtolower(explode('@', trim($email))[1])
+  return (isset($mail_domains_ko[$domain]) || array_key_exists($domain, $mail_domains_ko));
```

Comment:
Can be replaced to just `isset($mail_domains_ko[$domain])` instead of `isset($mail_domains_ko[$domain]) || array_key_exists($domain, $mail_domains_ko)`

Reply:
I feel it's always good practice to do that format: `(isset($mail_domains_ko[$domain]) || array_key_exists($domain, $mail_domains_ko))`. If `isset` does the job, the second part produces no overhead.

Reply:
In this case `array_key_exists($domain, $mail_domains_ko))` will never be called, that's dead code (nothing to do with good practice there).

Reply:
Let's merge this.
Can you also close your previous PR, as this one replaces the other (would have been better to do changes in the initial PR by the way).